### DISCUSSION TITLE
make a picture with pmtiles-show, any idea ?

### DISCUSSION THREAD

**Discussion by AnthonyHervy** [protomaps/PMTiles#110] [Q&A] [answered]
Hi, 

after several tries, my friend ChatGPT and I are unable to generate an image from a pmtiles file.

shp -> geojson -> mbtiles -> pmtiles, no problem

pmtiles-show found datas, but I don't know how to use them to generate a picture from that. 

![Capture d’écran 2023-01-04 à 17 58 06](https://user-images.githubusercontent.com/2773297/210608751-6480f6a5-b6b5-46bb-bbc8-4109eb9fcad9.png)

the ideal would be to be able to find the 8172 and 5863 values ​​from x y coordinates, but by digging in the python files I can't find how to do it

[17339.pmtiles.zip](https://github.com/protomaps/PMTiles/files/10345980/17339.pmtiles.zip)

if you can help, I offer coffee (or beer), i love pmtiles, so close to the point

edit : in pmtiles file, is pbf, not png

**Comment by bdon** [comment, marked as answer]
you need to redirect the output of pmtiles-show: `pmtiles-show 17339.pmtiles 14 8172 5863 > tile.mvt.gz` 

after you gunzip that file you have a vector tile on disk. but creating an "image" from that needs a tile renderer, which is a separate concern.

**Reply by AnthonyHervy** [reply]
thank you ! ok unlocked, with that I can continue, thank you ! if you have a link to support the project with pleasure to support
